btree/btree.c

```diff
@@ -6647,6 +6716,7 @@ static void ut_basic_kv_oper(void)
 		void                *k_ptr  = &key;
 
 		cred = M0_BE_TX_CB_CREDIT(0, 0, 0);
+		m0_btree_del_credit(tree, 1, ksize, -1, &cred);
```

Comment:
Passing -1 as value size is not available for delete operation.

Reply:
I agree but this might become a problem for us because we can never be able to get the proper credits since this will work for fixed format but might be an issue for variable value size as we discussed.

Reply:
Currently passing ksize, vsize param to node_free_credit function , in future when we implement node formats which require key, value size , we can use this param, but in delete routine, currently we are taking only key from user and not value, So we might want toc check if user is  going to provide whole record, this way we can get value size, else for credit calculation, we might want to store max value size somewhere.